__tests__/app.test.js

```diff
@@ -109,3 +107,38 @@ describe('GET /api/articles', () =>{
         });
     })
 })
+
+describe('GET /api/articles/:article_id/comments', () =>{
+    test('GET 200 returns an array of comments from an article ID sorted in descending order', () =>{
+        return request(app)
+        .get('/api/articles/5/comments')
+        .expect(200)
+        .then(({ body }) => {
+          const comments = body.article;
+          comments.forEach(comment => {
+            expect(comment).toHaveProperty('comment_id');
+            expect(comment).toHaveProperty('votes');
+            expect(comment).toHaveProperty('created_at');
+            expect(comment).toHaveProperty('author');
+            expect(comment).toHaveProperty('body');
+            expect(comment).toHaveProperty('article_id');
```

Comment:
Ideally we want to be asserting against the values as well here not just the keys to have as much coverage as possible. It's fine to assert against the type in each case except for the article_id as we know what that should be for every comment.